src/Components/ShareImageModal/RegionsSelect.js

```diff
@@ -43,7 +43,8 @@ export const selectRegionsToDisable = createSelector(
 const prepareRegions = (regionsToDisable) => {
   const regions = AWS_REGIONS.map((region) => ({
     ...region,
-    disabled: regionsToDisable.has(region.value),
+    disabled:
+      regionsToDisable.has(region.value) || region?.disableRegion === true,
```

Comment:
Could we add this logic to the `selectRegionsToDisable` selector (line 26) instead?

Reply:
This seems like it's going to be kind of tricky, we will refactor when we migrate to RTKQ.

Reply:
This is a bit tricky, let's keep it after the refactor